client/src/Card.js

```diff
@@ -0,0 +1,23 @@
+import "./Card.css";
+
+export const Card = ({
+  wrapperClass,
+  cardSubtitle,
+  cardTitle,
+  cardTextClass,
+  cardText,
+  cardLink,
+  cardLinkClass,
+  cardLinkText,
+}) => {
+  return (
+    <div className={wrapperClass}>
+      <p className="card-subtitle">{cardSubtitle}</p>
+      <h1 className="card-title">{cardTitle}</h1>
```

Comment:
There should only be one `h1` per page, so this should probably be a lower level heading?